EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -289,18 +290,54 @@ export default class EventsExpressService {
             method: "get",
             headers: new Headers({
                 'Content-Type': 'application/json',
-                'Authorization': `Bearer ${localStorage.getItem('token')}`
+                'Authorization': 'Bearer ' + localStorage.getItem('token')
             }),
         });
 
-        return !res.ok
-            ? {
+        if (res.ok) {
+            return await res.json();
+        }
+
+        else {
+            if (res.status === 401) {
+                let resUpdateToken = null;
+                localStorage.removeItem("token");
+
+                resUpdateToken = await fetch('api/token/refresh-token', {
+                    method: "POST"
+                });
+
+                if (!resUpdateToken.ok) {
+                    return this.getResourceResponseHandler(resUpdateToken);
+                }
+                let result = await this.getResourceResponseHandler(resUpdateToken);
+                localStorage.setItem('token', result.jwtToken);
+
+                return this.getResource(url);
+
+            }
+            else {
+                return {
+                    error: {
+                        ErrorCode: res.status,
+                        massage: await res.statusText
+                    }
+                }
+            }
```

Comment:
use 
if () { 
 ... 
} else if() { 
 ... 
} else {
 ... 
}

Reply:
did it